library/openssl/digest/shared/update.rb

```diff
@@ -0,0 +1,123 @@
+require_relative '../../../../library/digest/sha1/shared/constants'
+require_relative '../../../../library/digest/sha256/shared/constants'
+require_relative '../../../../library/digest/sha384/shared/constants'
+require_relative '../../../../library/digest/sha512/shared/constants'
+require 'openssl'
+
+describe :openssl_digest_update, shared: true do
+  context "it supports full data chunks" do
```

Comment:
What does `full data chunks` mean? As well as `smaller chunks`? It's OK if it's a common name/term, otherwise it's confusing.

Reply:
The concept is that you can keep adding data, so calling `update("abc")` yields the same result as calling `update("a"); update("b"); update("c)`, as does `update("ab"); update("c")`. I have no idea if chunk is the official word for this concept, but somehow that's how I would describe it.

I've updated the descriptions to:
```ruby
context "when given input as a single string"
context "when given input as multiple smaller substrings"
```
This is the most unambiguous term I could come up with, but I'm open to suggestions.

Reply:
👍